docs/reference/relay-workflows.md

```diff
@@ -80,7 +80,7 @@ Triggers map incoming events to workflows. The `triggers` key contains an array
 
 #### schedule
 
-A Schedule trigger type works, as the name suggests, on a time-based schedule. It needs a key named `schedule` whose value is a string in  the standard Unix 'cron' syntax. There's a handy [crontab syntax generator](https://crontab.guru/) that can help you build a schedule if you're not familiar with cron.
+A Schedule trigger type works, as the name suggests, on a time-based schedule. It needs a key named `schedule` whose value is a string in  the standard Unix 'cron' syntax. There's a handy [crontab syntax generator](https://crontab.guru/) that can help you build a schedule if you're not familiar with cron. To determine whether Relay supports a particular advanced cron syntax, check the docs for the underlying implementation at [github.com/robfig/cron](https://github.com/robfig/cron/blob/master/doc.go).
```

Comment:
May be best to link to the rendered godoc here: https://pkg.go.dev/github.com/robfig/cron/v3?tab=doc